src/skeletons/skeleton-text.tsx

```diff
@@ -0,0 +1,54 @@
+// https://www.figma.com/file/w7E0mmB92eio0zHw7h9iS2/%5BREADY%5D-Skeletons-Specs?node-id=986%3A1161
+
+import * as React from 'react';
+import Stack from '../stack';
+import {useIsInverseVariant} from '../theme-variant-context';
+import {useAnimation} from './styles';
+import {createUseStyles} from '../jss';
+
+type TextStylesProps = {
+    isInverse: boolean;
+};
+
+const useTextStyles = createUseStyles(({colors}) => ({
+    line: {
+        borderRadius: 8,
+        height: 8,
+        width: '100%',
+        background: ({isInverse}: TextStylesProps) =>
+            isInverse ? colors.backgroundSkeletonInverse : colors.backgroundSkeleton,
+    },
+    lastLine: {
+        width: '75%',
+    },
+}));
+
```

Comment:
https://ibit.ly/MmZD

I made this example, skeletons doesn't works with `<Inline>` component to make compositions like this
![image](https://user-images.githubusercontent.com/6722153/189305625-a1dd09fa-3bf4-4ab9-a29e-b14c457a6221.png)


For other side, do you think is interesting to add `width` prop to media skeleton to make skeleton buttons instead of wrap skeletons with box with custom width?

```
<Box width={150}>
	<SkeletonMedia ariaValueText="loading" height={48} />
</Box>
```

![image](https://user-images.githubusercontent.com/6722153/189306152-d6c263ae-29b1-4185-96ea-b9f40d242a80.png)

Reply:
And build for example row skeleton with two skeleton? like

```
<Inline space={16}>
	<SkeletonCircle />
	<SkeletonRect />
</Inline>
````

I think it is easier to build skeleton screens if you have some existing typology. I would keep SkeletonText and SkeletonRow as is. It is true that SkeletonMedia has a weird name if we want to use it as a button. Maybe rename SkeletonMedia to SkeletonRect? @aweell

Reply:
I would definitely keep `text`, but I'm not sure about `row`. Since the typology doesn't cover all existing components I suggest a more agnostic approach: `square`, `circle` ( and maybe `rounded?)` (With a `width` and `height` props and build the replacement with the layout components. 

My point is: Why the `row` should have a direct replacement and not other components? Based on usage? Is it better to have a list of components that should have skeleton version or something that allows building any component?

Reply:
Yes, Row Skeleton is because row design is spread over a bunch of screens

Reply:
This issue with skeletonRectangle inside an inline doesn't work
https://ibit.ly/DZlC

Reply:
you should place a defined width on the rectangle in this case, and you'll have the expected behavior

Reply:
https://mistica-mrgj5x2d2-tuentisre.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8BJAdgGwJZpgAgDMBXDDAdyygBcALPAZwAcBDMGAXmACYAGAXwB8AHTR48SAMoBrGBhhUIaAEowwVZmgDmckWLHMATlmYAZZgCNZ7ISAwRmUHJpu69NGFk00qnAIw9%2BVzEKahprEF4ANzIXUT1Hegs5AEE0LABbZiosRU4CZgx6GD5XAHphOMkZOQVlVXUtHTj9I1MLKxs7BydYvTx3T28-AJLmvBDacKiYkCC8BKSYVIysnLQ8gqLRsXK5qtl5RRU1DW0YOcNjM0sMcK7HLV63Dy8fYH9AsYmwm2mnsQW5hSaUy2VywHyhWKZQqfTE%2BxqR3qpyafUubRud3sD2cszGA1ew0%2BfW%2BUx40X%2B8ywiSBSxBq3BkK2MJESFK6GwuAEIAANCBaDB0jB6AgANogACyEEi1PUBhAAF0%2Bd8RfBRQBmABsPB5vi1AA4FXwgA

Reply:
https://mistica-mrgj5x2d2-tuentisre.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8BJAdgGwJZpgAgGcAHAQzBgF5gAmABgF88AzAVwwwHUsoAXACwB8AHTR48SAMoBrGBhg8IaAEowwPEmgDmckWLEkATlhIAZEgCNZFISAwQSUHJpu69fGFk18eVAIy0GVzFHAgs5AEE0LABbEh4sRSomEgwCGHpXAHphUXFpWXlFFTUNbRggvENjM0sMa1t7Ry0XXLF3T28-AIzWvBCwmEiYuIS0JJS0nrFskSRM9GxcHJAAGhB%2BGGiYAgQAbRAAWQgANywCdQMQAF01gHdufh34XYBmADZaFd93gA4r%2BiAA

Reply:
mmm in my opinion,  this way has a different behavior than the rest of the components and changes the way to build, but If @pladaria and @atabel I will ok